package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -513,6 +513,12 @@ class HydrogenBondLifetimes(object):
 
     def __init__(self, universe, selection1, selection2, t0, tf, dtmax,
                  nproc=1):
+        warnings.warn(
+            "This class is deprecated. "
```

Comment:
Need to add
- When it will be removed (2.0.0)
- .. deprecated reST to the text

(Have a look at other code how we normally deprecate.)